packages/rpc/package.json

```diff
@@ -29,6 +30,6 @@
     "postinstall-postinstall": "^2.1.0",
     "prettier": "^2.7.1",
     "typescript": "^5.1.3",
-    "vitest": "^0.32.0"
+    "vitest": "0.34.1"
```

Comment:
```suggestion
    "vitest": "^0.34.1"
```

Reply:
Done, but what is the benefit of this? Doesn't this make our build less deterministic?